func/v1/endpoints/ObligationsControllerISpec.scala

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.endpoints
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status._
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.WSRequest
+import support.IntegrationBaseSpec
+import v1.fixtures.ObligationsFixture
+import v1.models.errors._
+import v1.stubs.{AuditStub, AuthStub, DesStub}
+
+
+class ObligationsControllerISpec extends IntegrationBaseSpec with ObligationsFixture {
+
+  private trait Test {
+
+    val vrn: String = "123456789"
+    val fromDate: String = "2017-01-01"
+    val toDate: String = "2017-12-01"
+    val oblStatus: String = "F"
```

Comment:
missing a test for OK 200, with oblStatus:"O"

Reply:
Also a test for oblStatus:"", as per the spec: "Omit status to retrieve all obligations"